src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,25 @@
+.catalog {
+  display: flex;
+  justify-content: center;
+
+  &__cards {
+    display: flex;
+    flex-wrap: wrap;
+    padding: $catalog_cards-paddings-vertical 0;
+    margin-bottom: 72 - $catalog_cards-paddings-vertical;
```

Comment:
what is 72?

Reply:
![image](https://user-images.githubusercontent.com/60798246/180044246-a06f4c74-c6fb-4034-bd86-0161ae9bc55b.png)
![image](https://user-images.githubusercontent.com/60798246/180044621-c61e024b-b183-4892-ba87-7309d591928d.png)
what is 0?